pkg/controllers/management/feature/feature_handler.go

```diff
@@ -6,29 +6,37 @@ import (
 	"reflect"
 	"time"
 
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 
 	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
 	"github.com/rancher/rancher/pkg/auth/tokens"
+	datamanagement "github.com/rancher/rancher/pkg/data/management"
 	"github.com/rancher/rancher/pkg/features"
 	managementv3 "github.com/rancher/rancher/pkg/generated/controllers/management.cattle.io/v3"
+	normanv3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
+	"github.com/rancher/rancher/pkg/types/config"
 	"github.com/rancher/rancher/pkg/wrangler"
 )
 
 type handler struct {
 	featuresClient       managementv3.FeatureClient
 	tokensLister         managementv3.TokenCache
 	tokenEnqueue         func(string, time.Duration)
-	nodeDriverController managementv3.NodeDriverController
+	nodeDriverController normanv3.NodeDriverInterface
+	managementContext    *config.ManagementContext
 }
 
-func Register(ctx context.Context, wContext *wrangler.Context) {
+func Register(ctx context.Context, management *config.ManagementContext, wContext *wrangler.Context) {
 	h := handler{
 		featuresClient:       wContext.Mgmt.Feature(),
```

Comment:
I'd change this to a feature controller and enqueue instead of adding the driver here.

Reply:
I don't understand what you mean here, can you explain a bit more?